pkg/apis/eventing/v1alpha1/knativeeventing_types.go

```diff
@@ -64,6 +64,11 @@ type KnativeEventingSpec struct {
 	// If no registry is provided, the knative release images will be used.
 	// +optional
 	Registry Registry `json:"registry,omitempty"`
+
+	// The default broker type to use for the brokers Knative creates.
```

Comment:
The default broker class ?   I mean, that's what it is :-)

Reply:
happy to receive naming suggestions!